From 31b8caca9394150328e195fcf9ace6d16cb1f3d7 Mon Sep 17 00:00:00 2001
From: Vincent Knecht <vincent.knecht@mailoo.org>
Date: Sun, 21 Mar 2021 09:48:00 +0100
Subject: [PATCH 186/210] dt-bindings: power: supply: smb1360: Add
 qcom,otp-rslow-config

Document qcom,otp-rslow-config setting.

Signed-off-by: Vincent Knecht <vincent.knecht@mailoo.org>
---
 .../devicetree/bindings/power/supply/smb1360.yaml        | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Documentation/devicetree/bindings/power/supply/smb1360.yaml b/Documentation/devicetree/bindings/power/supply/smb1360.yaml
index b94c98e03e3..a90eede1fab 100644
--- a/Documentation/devicetree/bindings/power/supply/smb1360.yaml
+++ b/Documentation/devicetree/bindings/power/supply/smb1360.yaml
@@ -188,6 +188,15 @@ properties:
       If this value is not specified a default value of 50mV is used.
       Unit is in milli-volts.
 
+  qcom,otp-rslow-config:
+    $ref: /schemas/types.yaml#/definitions/uint8-array
+    description:
+      Undocumented feature found on some devices as "qcom,otp-rslow-cfg".
+      Each item is a value keyed respectively to 0x54, 0x55, 0x56 and 0x57
+      and written to OTP.
+    minItems: 4
+    maxItems: 4
+
   usb_otg_vbus:
     description: Regulator controlling USB OTG VBUS.
     # TODO: Reference regulator-max-microamps here somehow?
-- 
2.34.1

